README.md

```diff
@@ -19,6 +19,10 @@ LMAX Development Team
 
 ## Changelog
 
+### 3.4.3
```

Comment:
This change is definitely NOT a minor point release. It would break compatibility with 99% of users of this library.
Revert the changelog change as this will not go out in a release any time soon, not until 4.0